Makefile

```diff
@@ -7,6 +7,13 @@ GOSRC=$(GOPATH)/src
 
 u := $(if $(update),-u)
 
+$(GOBIN)/wire:
+	GO111MODULE=off go get github.com/google/wire/cmd/wire
+
+.PHONY: wire
+wire: $(GOBIN)/golangci-lint
```

Comment:
Shouldn't it depend on wire install goal? It fails on my local

```
wire gen ./...
make: wire: No such file or directory
make: *** [wire] Error 1
```

Reply:
I'm so sorry... I fixed it. Would you try it again?

Reply:
Yup, it works now